src/components/pages/adminHome/index.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import '../../../styles/InstructorStyles/index.less';
-import AdminSidebar from './adminSidebar';
+import AdminSidebar from './AdminSidebar';
 import AdminCalendar from './adminCalendar';
```

Comment:
I'll make a trello ticket for this since the changes here are breaking the code rather than the rest of the names which need to get fixed for consistency